Marlin/src/libs/MAX31865.cpp

```diff
@@ -150,17 +152,16 @@ void MAX31865::begin(max31865_numwires_t wires, float zero, float ref) {
   Rzero = zero;
   Rref = ref;
 
-  OUT_WRITE(_cs, HIGH);
+  SET_OUTPUT(_cs);
+  write(_cs, HIGH);
```

Comment:
This change should be reverted. The macro `OUT_WRITE()` acheives both of these. In this case, the local `write()` is unneccesary.

Reply:
I created a wrapper write function that writes signals to the pin according to the SPI speed. Not sure if this is an important step but I've had pretty good results in terms of stability of my max31865 with my SKR E3 turbo board

Reply:
@slowbro — Note that FastIO macros like `WRITE` and `OUT_WRITE` require the first argument to be known at compile-time. They cannot take a variable, only a pin name or equivalent preprocessor define. If the pin is a variable, then the code must use `pinMode`, `digital_write`, etc.